src/hotspot/os/windows/os_windows.cpp

```diff
@@ -3215,7 +3215,7 @@ static char* allocate_pages_individually(size_t bytes, char* addr, DWORD flags,
   if ((flags & MEM_COMMIT) != 0) {
     MemTracker::record_virtual_memory_reserve_and_commit((address)p_buf, bytes, CALLER_PC);
   } else {
-    MemTracker::record_virtual_memory_reserve((address)p_buf, bytes, CALLER_PC);
+    MemTracker::record_virtual_memory_reserve((address)p_buf, bytes, CALLER_PC, mtNone);
```

Comment:
Use the correct MEMFLAG here instead of mtNone.

Reply:
Done.